tests/functests/instancetype_test.go

```diff
@@ -26,6 +26,12 @@ const (
 	sshPort = 22
 )
 
+var skipPreference = map[string]any{
```

Comment:
Move that to the resource requirements context?

Reply:
Done.